locales/en/translation.json

```diff
@@ -122,6 +122,7 @@
   "Parameter": "Setting",
   "Settings on day": "Settings on {{day}}",
   "Parameters History": "Settings History",
+  "Pen Bolus": "Pen",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
  "Pen": "Pen",
```